src/prpy/ik_ranking.py

```diff
@@ -53,13 +53,26 @@ def JointLimitAvoidance(robot, ik_solutions):
 
 
 class NominalConfiguration(object):
-    def __init__(self, q_nominal):
+    def __init__(self, q_nominal, max_deviation=2*numpy.pi):
```

Comment:
This is exactly the case I was referring to:

> The only special case where you might need explicit multirotation is where the intermediate trajectory needs to wrap a full rotation to avoid an intermediate obstacle that partitions the configuration space 

However, I didn't think about the combinatorial effect that this could have when there are multiple multi-revolution joints. I buy your argument now. :smile: